package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.79.4",
+  "version": "3.0.0-epilson",
```

Comment:
Shouldn't have the `-epsilon` suffix.

Reply:
The suffix will be dropped prior to release to Production. For now, it's there only because I've not created a release yet - just a tag for QA to test.